.github/terraform/aws/main.tf

```diff
@@ -109,11 +148,21 @@ resource "aws_security_group" "sg" {
   }
 }
 
+resource "tls_private_key" "ssh" {
+  algorithm = "RSA"
+  rsa_bits  = "4096"
+}
+
+resource "local_file" "private_key" {
```

Comment:
Not sure if we need the local file anymore since everything is handled in Terraform. Is it for debugging?

Reply:
Exactly. In case there is fail with this key it is easy to login and debug error on the machine. 

We could add some kind of a contrition to only create this file if needed if you want?